x/assets/keeper/bank.go

```diff
@@ -36,6 +38,14 @@ func (k Keeper) PerformDepositOrWithdraw(ctx sdk.Context, params *DepositWithdra
 		return errorsmod.Wrapf(assetstypes.ErrInvalidOperationType, "the operation type is: %v", params.Action)
 	}
 
+	if assetstypes.IsNativeToken(assetID) {
```

Comment:
This is a bit confusing. We should use `NST` to distinguish with native tokens from #127 or maybe say native restaked token?

Reply:
updated